woocommerce/woocommerce-ios
# Login Onboarding: analytics

Part of #7284 


### Description


This PR implements the tracking events as in pe5sF9-dj-p2#comment-376:

event | trigger | properties
-- | -- | --
`*_login_onboarding_shown` | When login onboarding is shown in the app. | n/a
`*_login_onboarding_next_button_tapped` | When the user taps on the “Next” button on the login onboarding screen to see the next app feature or continue to the login prologue screen. | is_final_page: Bool whether the user taps the “Next” button on the final page of the feature.
`*_login_onboarding_skip_button_tapped` | When the user taps on the “Skip” button on the login onboarding screen to enter the prologue screen. | n/a

### Testing instructions


Prerequisite: you can manually return `false` in `LoggedOutAppSettings.hasFinishedOnboarding` to always show login onboarding for testing this PR.

- Launch the app
- Log out if needed --> login onboarding screen should be shown with an event logged: `login_onboarding_shown`
- Tap "Next" only twice to go to the last feature page on the onboarding screen so that it is not dismissed --> there should be 2 events logged: `login_onboarding_next_button_tapped` with `is_final_page: false`
- Tap "Next" again --> it should go to the prologue screen with an event logged: `login_onboarding_next_button_tapped` with `is_final_page: true`
- Relaunch the app --> login onboarding screen should be shown again with an event logged: `login_onboarding_shown`
- Tap "Skip" --> it should go to the prologue screen with an event logged: `login_onboarding_skip_button_tapped`


---
